openpype/hosts/blender/plugins/publish/collect_instances.py

```diff
@@ -19,85 +20,66 @@ class CollectInstances(pyblish.api.ContextPlugin):
 
     @staticmethod
     def get_asset_groups() -> Generator:
-        """Return all 'model' collections.
-
-        Check if the family is 'model' and if it doesn't have the
-        representation set. If the representation is set, it is a loaded model
-        and we don't want to publish it.
+        """Return all instances that are empty objects asset groups.
         """
         instances = bpy.data.collections.get(AVALON_INSTANCES)
         for obj in instances.objects:
-            avalon_prop = obj.get(AVALON_PROPERTY) or dict()
+            avalon_prop = obj.get(AVALON_PROPERTY) or {}
             if avalon_prop.get('id') == 'pyblish.avalon.instance':
                 yield obj
 
     @staticmethod
     def get_collections() -> Generator:
-        """Return all 'model' collections.
-
-        Check if the family is 'model' and if it doesn't have the
-        representation set. If the representation is set, it is a loaded model
-        and we don't want to publish it.
+        """Return all instances that are collections.
         """
-        for collection in bpy.data.collections:
-            avalon_prop = collection.get(AVALON_PROPERTY) or dict()
+        instances = bpy.data.collections.get(AVALON_INSTANCES)
+        for collection in instances.children:
+            avalon_prop = collection.get(AVALON_PROPERTY) or {}
             if avalon_prop.get('id') == 'pyblish.avalon.instance':
                 yield collection
 
```

Comment:
Couldn't this just `return CollectInstances.get_asset_groups()` - but even better - why do we need both staticmethods if they are the same?

Reply:
Yeah you're right! I started reworking the collector with the idea to keep it with a similar structure, but ended changing the `process` method to be in a single loop, and in the end kept the two functions to get the instances 😅 I'll change them!